pkg/kubelet/cm/helpers_linux.go

```diff
@@ -44,16 +44,17 @@ const (
 	SharesPerCPU  = 1024
 	MilliCPUToCPU = 1000
 
-	// 100000 is equivalent to 100usec
-	QuotaPeriod    = 100000
+	// 100000 microseconds is equivalent to 100ms
+	QuotaPeriod = 100000
+	// 1000 microseconds is equivalent to 1ms
 	MinQuotaPeriod = 1000
```

Comment:
Let's add a link to the kernel source where min is defined:

https://github.com/torvalds/linux/blob/cac03ac368fabff0122853de2422d4e17a32de08/kernel/sched/core.c#L10546